node_utils.js

```diff
@@ -45,7 +45,9 @@ function get_conf() {
 function get_redis_subscriber(kind = "redis_queue", options = {}) {
 	const conf = get_conf();
 	const host = conf[kind];
-	return redis.createClient({ url: host, ...options });
+	const client = redis.createClient({ legacyMode: true, url: host, ...options });
+	client.connect();
```

Comment:
This gets stuck in infinite retry if not available. 

![image](https://github.com/frappe/frappe/assets/9079960/c660a5bb-6cbe-47a0-a4a8-ff722516f974)

Reply:
It turned out that non-legacy mode (i.e. full migration) doesn't suffer from this https://github.com/frappe/frappe/pull/21797